<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackLayoutTableWidget</class>
 <widget class="QWidget" name="TrackLayoutTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>758</width>
        <height>199</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="checkTabVerticalLayout">
         <item>
          <widget class="QLabel" name="checkTabTableLabel">
           <property name="text">
            <string>Select logs to display in track numbers</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="chkboxTableWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="selectionGroupBox">
       <property name="title">
        <string>Data selection</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="selectDefaultRadioButton">
          <property name="toolTip">
           <string>Default log types to plot can be set in preferences</string>
          </property>
          <property name="text">
           <string>Select default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="selectActiveRadioButton">
          <property name="toolTip">
           <string>Plot active logs</string>
          </property>
          <property name="text">
           <string>Select active</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="selectAllRadioButton">
          <property name="text">
           <string>Select all</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="selectNoneRadioButton">
          <property name="text">
           <string>Select none</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="miscGroupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="addTrackPushButton">
          <property name="text">
           <string>Add track</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
